### PR TITLE
BH-664 Remove MySQL port translation in CI

### PR DESCRIPTION
**Pull request by BitOne**
**Description (for Contributor and Core Developer)**

Remove MySQL port translation that fails intermittently during the CI.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by SamirBoulil — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,6 +38,15 @@ jobs:
           command: |
             sed -i "s|\"akeneo/pim-community-dev\": \"dev-master|\"akeneo/pim-community-dev\": \"dev-${CIRCLE_BRANCH}|" composer.json
             sed -i "s|\"akeneo/pim-community-dev\": \"dev-master|\"akeneo/pim-community-dev\": \"dev-${CIRCLE_BRANCH}|" grth/composer.json
+      - run:
+          name: Install yq
+          command: |
+            wget https://github.com/mikefarah/yq/releases/download/3.3.1/yq_linux_386
+            sudo mv yq_linux_386 /usr/local/bin/yq
+            sudo chmod +x /usr/local/bin/yq
```
Comment (vgelot) [inline .circleci/config.yml:46]:
```suggestion
      - install_yq
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -53,6 +62,15 @@ jobs:
       image: ubuntu-2004:202101-01
     steps:
       - checkout
+      - run:
+          name: Install yq
+          command: |
+            wget https://github.com/mikefarah/yq/releases/download/3.3.1/yq_linux_386
+            sudo mv yq_linux_386 /usr/local/bin/yq
+            sudo chmod +x /usr/local/bin/yq
```
Comment (vgelot) [inline .circleci/config.yml:70]:
```suggestion
      - install_yq
```

**Review by vgelot — APPROVED** [review]
Add need to create the command at the buttom of the file
```install_yq:
    description: "Install yq"
    steps:
      - run:
          name: Install yq
          command: |
            wget https://github.com/mikefarah/yq/releases/download/3.3.1/yq_linux_386
            sudo mv yq_linux_386 /usr/local/bin/yq
            sudo chmod +x /usr/local/bin/yq```

**Review by AymericPlanche — APPROVED** [review]

**Review by adesaegher — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (BitOne) [reply .circleci/config.yml]: Done!

File: .circleci/config.yml (hunk 2 above)
Comment (BitOne) [reply .circleci/config.yml]: Done!

**Pull request merged**